packages/portfolio/src/adapters/dullahan-api/api-maker.ts

```diff
@@ -14,11 +14,13 @@ import {
   toProcessedMediaRequest,
   toSecondaryTokenInfos,
   toTokenActivityUpdates,
```

Comment:
can you refactor to `toTokenActivity`, the Updates doesn't make sense anymore. I forgot to do it :/